packages/echo-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@equinor/echo-core",
-    "version": "0.3.25",
+    "version": "0.3.26",
```

Comment:
I think you can leave 0.3.25 as we didn't made a new release yet with that number.
From npmjs:
![image](https://user-images.githubusercontent.com/13872908/138039217-1ca4f7a6-ff73-49c2-a47e-af85803f585a.png)